<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-03T21:30:18 IST" name="Suite" finished-at="2022-09-03T21:31:48 IST" duration-ms="90396">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationsTest.LoginErrorValidation()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" name="LoginErrorValidation" class="AtechCompany.tests.ErrorValidationsTest"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" name="submitOrder" class="AtechCompany.tests.SubmitOrderTest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2022-09-03T21:30:18 IST" name="Submit Order Test" finished-at="2022-09-03T21:31:47 IST" duration-ms="89316">
      <class name="AtechCompany.tests.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:30:18 IST" name="launchApplication" finished-at="2022-09-03T21:31:08 IST" duration-ms="49601" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:08 IST" name="submitOrder" data-provider="getData" finished-at="2022-09-03T21:31:19 IST" duration-ms="11129" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=RAM@k123, product=ZARA COAT 3, email=ram.kumar@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:19 IST" name="tearDown" finished-at="2022-09-03T21:31:20 IST" duration-ms="930" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:20 IST" name="launchApplication" finished-at="2022-09-03T21:31:30 IST" duration-ms="10732" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:30 IST" name="submitOrder" data-provider="getData" finished-at="2022-09-03T21:31:35 IST" duration-ms="5029" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=PANKAJ@k123, product=ADIDAS ORIGINAL, email=pankaj.kumar@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:35 IST" name="tearDown" finished-at="2022-09-03T21:31:36 IST" duration-ms="116" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:36 IST" name="launchApplication" finished-at="2022-09-03T21:31:45 IST" duration-ms="9710" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:45 IST" name="OrderHistoryTest" finished-at="2022-09-03T21:31:47 IST" depends-on-methods="AtechCompany.tests.SubmitOrderTest.submitOrder" duration-ms="1370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AtechCompany.tests.SubmitOrderTest@624ea235]" started-at="2022-09-03T21:31:47 IST" name="tearDown" finished-at="2022-09-03T21:31:47 IST" duration-ms="334" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- AtechCompany.tests.SubmitOrderTest -->
    </test> <!-- Submit Order Test -->
    <test started-at="2022-09-03T21:30:18 IST" name="Error Validation Test" finished-at="2022-09-03T21:31:48 IST" duration-ms="90396">
      <class name="AtechCompany.tests.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:30:18 IST" name="launchApplication" finished-at="2022-09-03T21:31:08 IST" duration-ms="50279" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:08 IST" name="LoginErrorValidation" finished-at="2022-09-03T21:31:14 IST" duration-ms="5793" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at AtechCompany.tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:14 IST" name="tearDown" finished-at="2022-09-03T21:31:16 IST" duration-ms="2508" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:16 IST" name="launchApplication" finished-at="2022-09-03T21:31:30 IST" duration-ms="14081" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:30 IST" name="LoginErrorValidation" finished-at="2022-09-03T21:31:31 IST" duration-ms="776" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at AtechCompany.tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:33 IST" name="tearDown" finished-at="2022-09-03T21:31:33 IST" duration-ms="198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:33 IST" name="launchApplication" finished-at="2022-09-03T21:31:45 IST" duration-ms="11934" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:45 IST" name="ProductErrorValidation" finished-at="2022-09-03T21:31:48 IST" duration-ms="3049" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AtechCompany.tests.ErrorValidationsTest@7068e664]" started-at="2022-09-03T21:31:48 IST" name="tearDown" finished-at="2022-09-03T21:31:48 IST" duration-ms="361" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- AtechCompany.tests.ErrorValidationsTest -->
    </test> <!-- Error Validation Test -->
  </suite> <!-- Suite -->
</testng-results>
